src/ts/ConvivaAnalytics.ts

```diff
@@ -189,6 +196,7 @@ export class ConvivaAnalytics {
 
```

Comment:
_I can't comment on the documentation of this method._

Please add a remark to the method documentation that no session will be created automatically anymore after calling this.

Reply:
Thanks, added a note to the `endSession` method documentation